lib/saml11.js

```diff
@@ -42,10 +42,11 @@ exports.create = function(options, callback) {
                   algorithms.digest[options.digestAlgorithm]);
 
   sig.signingKey = options.key;
-  
+
   sig.keyInfoProvider = {
     getKeyInfo: function () {
-      return "<X509Data><X509Certificate>" + cert + "</X509Certificate></X509Data>";
+      var prefix = options.prefix ? options.prefix + ':' : '';
+      return "<" + prefix + "X509Data><" + prefix + "X509Certificate>" + cert + "</" + prefix + "X509Certificate></" + prefix + "X509Data>";
```

Comment:
please use `util.format` (it's more clear)

https://millermedeiros.github.io/mdoc/examples/node_api/doc/util.html#util.format